### PR TITLE
Add code owners for missing and updated sections

### PR DESCRIPTION
**Pull request by alexearnshaw**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Noticed some new sections/updated sections were not listed in codeowners so this PR is to fix that.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -49,37 +50,43 @@
...
+/content/en/docs/apim_policydev/apigw_gw_instances @alexearnshaw
 /content/en/docs/apim_policydev/apigw_kps/ @andreamussap
 /content/en/docs/apim_policydev/apigw_oauth/ @alexearnshaw
-/content/en/docs/apim_policydev/apigw_polref/ @andreamussap
 /content/en/docs/apim_policydev/apigw_poldev/ @alexearnshaw
+/content/en/docs/apim_policydev/apigw_polref/ @andreamussap
+/content/en/docs/apim_policydev/apigw_web_services @andreamussap
 /content/en/docs/apim_reference/ @andreamussap
 /content/en/docs/apim_relnotes/*apimgr_relnotes.md @alexearnshaw
 /content/en/docs/apim_relnotes/*apip_relnotes.md @andreamussap
```
Comment (andreamussap) [inline .github/CODEOWNERS:78]: @alexearnshaw  is this correct? An * before the name of the file?

/content/en/docs/apim_relnotes/***apip_relnotes.md**

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -49,37 +50,43 @@
...
+/content/en/docs/apim_administration/apimgr_sso/ @alexearnshaw
 /content/en/docs/apim_administration/apiportal_admin/ @andreamussap
+/content/en/docs/apim_administration/apiportal_sso/ @andreamussap
+/content/en/docs/apim_howto_guides/ @alexearnshaw
 /content/en/docs/apim_installation/apigtw_install/ @alexearnshaw
 /content/en/docs/apim_installation/apigw_containers/ @alexearnshaw
 /content/en/docs/apim_installation/apigw_upgrade/ @alexearnshaw
 /content/en/docs/apim_installation/apiportal_docker/ @andreamussap
 /content/en/docs/apim_installation/apiportal_install/ @andreamussap
+/content/en/docs/apim_policydev/apigw_external_connections @alexearnshaw
```
Comment (andreamussap) [inline .github/CODEOWNERS:69]: almost all paths finish with a slash (/). Does it works if there's no slash in the end?

Comment (alexearnshaw) [reply .github/CODEOWNERS]: good spot, thanks, fixed now

File: .github/CODEOWNERS (hunk 1 above)
Comment (alexearnshaw) [reply .github/CODEOWNERS:78]: This is correct as it matches on a file with any date where the * is

**Pull request merged**